README.md

```diff
@@ -30,36 +32,36 @@ Im Body wird eine Payload als JSON in folgendem Format verschickt:
+  Because of the [limits for Topics](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#iot-protocol-limits) to a maximium of 7 it can happen that a topic gets cut off if the corresponding Partner tree is too deep. To still be able to notify the subscriber about a Kundenbetreuer, the property `kundenbetreuerPartnerbaum` contains the full tree.
 
 - `letztesAenderungsDatum`:
 
-  Das übermittelte Änderungsdatum entspricht dem aktuellen Zeitstempel beim Versand der Message.
+  This property is always the current timestamp at the time the message was sent.
 
 - `quellsystem`:
 
-  Derzeit wird als Quellsystem stets `KREDITSMART` eingetragen. Das Attribut ist für zukünftige Erweiterungen über KreditSmart hinaus vorgesehen.
+  Currently only `KREDITSMART` is supported as a possible source. This property is added as a preparation for future expansions.
 
-## Auslöser und Frequenz von Push-Nachrichten
+## Trigger and frequency of push notifications
 
-Eine Nachricht wird unabhängig von der Art der Änderung eines Vorgangs oder eines Antrags verschickt. Je nach Aktivität am Vorgang ergibt sich daraus eine entsprechende Frequenz an Push-Nachrichten.
+A message will be publish regardless if the Vorgang or Antrag is updated. Therefore the activity in the Vorgang is responsible for the frequency of push notifications.
 
-Wir aggregieren derzeit keine Nachrichten, d.h. Clients sind selbst dafür verantwortlich die über AWS potentiell eingehende Last zu verarbeiten.
+Currently we do not aggregate any messages, that means the client is responsible for processing the potential load of messages.
 
-In KreditSmart werden keine Push-Nachrichten aufbewahrt. Daher gehen Nachrichten in den Zeiträumen verloren, in denen kein Subscriber für die jeweiligen Topics mit AWS verbunden ist.
+KreditSmart does not save any push notifications. That means if there is no history of messages in intervals a subscriber is not connected to the topics.
 
-## Client-Implementierung
+## Client-Implementation
 
-Für die Implementierung und Konfiguration eines MQTT Clients gibt es eine Vielzahl von Beispiel-Implementierungen.
-AWS bietet selbst ein passendes Java-SDK an, das unter [github.com/aws/aws-iot-device-sdk-java](https://github.com/aws/aws-iot-device-sdk-java#use-the-sdk) inklusive Beispiel-Code dokumentiert ist.
+There are a numver of different examples for how to implement and configure a MQTT client.
+AWS provides a Java-SDK which is documented under [github.com/aws/aws-iot-device-sdk-java](https://github.com/aws/aws-iot-device-sdk-java#use-the-sdk) including sample code.
 
-## Nutzungsbedingungen
-Die APIs werden unter folgenden [Nutzungsbedingungen](https://docs.api.europace.de/nutzungsbedingungen/) zur Verfügung gestellt
+## Terms of use
+The APIs are made available under the following [Terms of Use](https://docs.api.europace.de/terms/).
```

Comment:
```suggestion
The APIs are made available under our [Terms of Use](https://docs.api.europace.de/terms/).
```

Reply:
Diesen Text möchte ich auch ungern anfassen, da wir den schon anderweitig verwenden

Reply:
Mh. Für mich suggeriert das "the following", dass ein Doppelpunkt oder Ähnliches kommen müsste, also noch etwas _nach_ "Terms of Use".